src/silicone/database_crunchers/ssp_specific_relation.py

```diff
@@ -2,13 +2,10 @@
 Module for the database cruncher which makes a linear interpolator from a subset of scenarios
 """
 
-import numpy as np
-import pandas as pd
-import scipy.interpolate
 from pyam import IamDataFrame
 
 from .base import _DatabaseCruncher
-from ..utils import _get_unit_of_variable
+from ..utils import _get_unit_of_variable, _make_wide_db, make_interpolator
```

Comment:
`_make_interpolator` and `_make_wide_db` are specific to this cruncher aren't they? Hence they should stay in this module?

Reply:
They are also used in the find_closest code, hence I moved it to go with them